queries/scheme/highlights.scm

```diff
@@ -0,0 +1,181 @@
+;; A highlight query can override the highlights queries before it.
+;; So the order is important.
+;; We should highlight general rules, then highlight special forms.
+
+(number) @number
+(character) @character
+(boolean) @boolean
+(string) @string
+(comment) @comment
+
+;; highlight for datum comment
+;; copied from ../clojure/highlights.scm
+([(comment) (directive)] @comment
```

Comment:
is the priority really necessary? For clojure it was only needed because it had s-expressions that could be comments and the parser would parse data within that special comment-sexpr.

Reply:
Yes. In Scheme R6RS, there is datum comment `#;` which comment the next datum:

```scheme
(list #;(1 2) 2)
; => (2)
```

where datum means a subset of Scheme object: number, string, list, vector, etc.
I have to add priority to make the whole datum comment highlight as normal comment.
I've tried putting the rule at the end of the file, but it doesn't work.